tox.ini

```diff
@@ -3,6 +3,7 @@ envlist =
     lint,
     test,
     check-migrations,
+    type,
```

Comment:
Maybe run this before the `test` step, since it should be a bit faster and produce more obvious errors.

Reply:
:+1: